### DISCUSSION TITLE
Why my website don't get traffic?

### DISCUSSION THREAD

**Discussion by RajaAbdullah302** [community/community#27690] [Other Feature Feedback, Questions, & Ideas]
Im a SEO begginer .  Im working on a website which provide food for there coustmer. But It can not get traffic. IDK why.
If any of  you are avalible then please help me. https://prontopizzaandrestaurant.com/

**Comment by radar-blip** [comment]
Yeah, I'll have the BBQ Chicken Pan Pizza please. Thanks!

Anyway, How long has this website been up and running? 
Have you done any search engine optimization?

**Comment by aayushchugh** [comment]
Your website is on 3rd number in google search so that's not an issue for me.

For attracting users to your website you have to do some marketing so that users
can know that your website even exists. 📈
So you should spend some money  on marketing 💸

**Comment by sarah-shaelynn** [comment]
hi, I'm getting the same issue at (https://duplicatebills.com/) as I'm doing all things clear.

**Reply by aayushchugh** [reply]
I think you should spend some time on UI of website

**Comment by varma59** [comment]
Submit your website to Google, Bing Serach engine tools and add meta tags to your website including keywords, Description of website.
